docs/source/installation.rst

```diff
@@ -0,0 +1,132 @@
+.. _usr_installation:
+
+Installation
+############
+
+Downloading Code
+================
+
+The latest code can be obtained through `theochem <https://github.com/theochem/AtomDB/>`_ in Github,
+
+.. code-block:: bash
+
+   git clone https://github.com/theochem/AtomDB.git
+
+.. _usr_py_depend:
+
+
+Installation
+============
+
+Installation via pip can be done by the following command:
+
+
+.. code-block:: bash
+
+    pip install git+https://github.com/theochem/AtomDB.git
+
+
+AtomDB can also be installed by cloning via git,
+
+
+.. code-block:: bash
+
+   git clone https://github.com/theochem/AtomDB.git
+
+
+Then installation via pip can be done by going into the directory where AtomDB was downloaded to
+and running,
+
+
+.. code-block:: bash
```

Comment:
This is just a recomentadion and it is mainly pertinent to people who may want to contribute to the package:
And section for installation inside conda venv
Add code snippet to show installation in editable mode 

For example I like the Istallation instructions section from [ChemTools](https://chemtools.org/usr_doc_installation.html)

Reply:
I added the editable installation.